app/components/contact/contact-item-modal.js

```diff
@@ -0,0 +1,28 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class ContactContactItemModalComponent extends Component {
+  @tracked contact = {};
```

Comment:
Ik zou deze component al laten werken met ember-data records i.p.v. een plain JS object. Hierdoor gaat deze modal later herbruikbaar zijn om ook contact-items te bewerken die reeds opgeslagen zijn.

Je kan, net zoals bij bronnen, voor ieder contact-item onmiddellijk een telephone en mail record aanmaken. Indien er niets ingevuld wordt in deze invulvelden, blijft het `value` attribuut hiervan gewoon leeg.

Het aanmaken/verwijderen/opslaan zou ik, zoals je nu gedaan hebt, buiten deze component houden zodat deze component als enige verantwoordelijkheid heeft dat de gegevens aangepast kunnen worden.